### ISSUE TITLE
Realtime audio flag set to null

### ISSUE THREAD

**Issue by boonier** [rorywalsh/csoundo#3]
Hi @rorywalsh 

All the examples seem to have `-+rtaudio=null` which prevents them from making sound. 

Might be confusing to some :)

**Comment by rorywalsh** [comment]
Agreed, I think they were added when we tried to get everything working in the, then experimental, Android exporter. The whole repo could do with updating but I rarely use this anymore now that the Csound WASM build is out.
